commands/utils/consts.go

```diff
@@ -105,6 +106,8 @@ const (
 	// Frogbot Git author details showed in commits
 	frogbotAuthorName  = "JFrog-Frogbot"
 	frogbotAuthorEmail = "eco-system+frogbot@jfrog.com"
+
+	AggregateFixesDefaultValue = false
```

Comment:
In my opinion it's unnecessary. When you initialize a struct, the default value is false. Also called `Zero value`:
https://go.dev/tour/basics/12#:~:text=The%20zero%20value%20is%3A,the%20empty%20string)%20for%20strings.